jellyfin_kodi/objects/movies.py

```diff
@@ -176,6 +176,11 @@ def get_path_filename(self, obj):
                 raise Exception("Failed to validate path. User stopped.")
 
             obj['Path'] = obj['Path'].replace(obj['Filename'], "")
+            '''check bluray directries and point it to ./BDMV/index.bdmv'''
+            if validate_bluray_dir(obj['Path'] + obj['Filename']):
+                obj['Path'] = obj['Path'] + obj['Filename'] + '/BDMV/'
+                obj['Filename'] = 'index.bdmv'
+                LOG.debug("path redirect %s",obj['Path'])
```

Comment:
Maybe change the log entry to something like
```suggestion
                LOG.debug("Bluray directory %s",obj['Path'])
```
Just so it's a bit more descriptive